src/state/inviteListProject/inviteListProjectSlice.js

```diff
@@ -0,0 +1,41 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { publicGet } from "../../utilities";
+export const inviteListProject = createAsyncThunk(
+  "inviteListProjectSlice/inviteListProject",
+  async (projectId) => {
+    try {
+      const response = await publicGet(
+        `/project/invites/62e90a9b4b46ed258587ef7f`
```

Comment:
Why hard coded id? Remove. It shall be there as params.

Reply:
updated